source/module_esolver/esolver_ks_lcao.h

```diff
@@ -78,6 +81,8 @@ class ESolver_KS_LCAO : public ESolver_KS<TK> {
 
     TwoCenterBundle two_center_bundle_;
 
+    rdmft::RDMFT<TK, TR> rdmft_solver;  // added by jghan for rdmft calculation, 2024-03-16
```

Comment:
I would suggest not adding any data in esolver_ks_lcao.h

Reply:
Sorry that I cannot find anywhere better to put this object, because its life span is a whole ion-step. Do you have any suggestions?

Reply:
I would not suggest writing code in this way, because: for example, if the initialization of rdmft causes a segmentation fault, it will affect the original LCAO developers. A tmp solution may be: inherit esolver_ks_lcao and generate a new esolver: esolver_ks_rdmft class, then you can add this variable in that esolver_ks_rdmft? In that case, the original LCAO will not depend on any details about rdmft.

Reply:
Therefore, the rdmft can be a esolver, it is fine rdmft needs esolver_ks_lcao because rdmft inherits esolver_ks_lcao

Reply:
I agree, but not this PR (it is just a initial-guess generation).  
The true rdmft esolver will come in with the optimization algirithm in the future, which will inherit `ESolver_KS_LCAO`.